File: apps/campo-android/app/src/main/java/com/arles/viverocampo/data/StaticFoundationRepository.kt
package com.arles.viverocampo.data

import com.arles.viverocampo.core.TechnicalIdentifiers
import com.arles.viverocampo.domain.FoundationRepository
import com.arles.viverocampo.domain.FoundationStatus

class StaticFoundationRepository : FoundationRepository {
    override fun currentStatus() = FoundationStatus(
        title = "Vivero Campo",
        message = "Fundación técnica instalada",
        firebaseStatus = TechnicalIdentifiers.FIREBASE_STATUS,
    )
}

File: apps/maestro-desktop/src/data/StaticFoundationRepository.ts
import { TECHNICAL_CONFIG } from "../core/technicalConfig";
import type { FoundationRepository } from "../domain/FoundationRepository";
import type { FoundationStatus } from "../domain/FoundationStatus";

export class StaticFoundationRepository implements FoundationRepository {
  currentStatus(): FoundationStatus {
    return {
      title: "Vivero Maestro",
      message: "Fundación técnica instalada",
      firebaseStatus:
        window.viveroFoundation?.getRuntimeStatus() ?? TECHNICAL_CONFIG.firebaseStatus,
    };
  }
}
